src/Drupal/Driver/DrushDriver.php

```diff
@@ -82,6 +83,7 @@ public function __construct($alias = NULL, $root_path = NULL, $binary = 'drush',
 
     $this->binary = $binary;
 
+    // @todo This should not be optional but injected by a factory method.
     if (!isset($random)) {
       $random = new Random();
```

Comment:
This won't actually work as written now, since `Random` won't be available without a `use` statement. I'd be happy to change this to using a factory method if you want to take a go at that.